pages/index.tsx

```diff
@@ -1,9 +1,26 @@
+import CodeBlock from '@/components/CodeBlock';
 import { signOut, useSession } from 'next-auth/react';
 import { useRouter } from 'next/router';
+import React from 'react';
```

Comment:
```suggestion
import { useState } from 'react';
```
마찬가지로 useState로 바꾸는게 어떨까용